src/src/Globals/Plugins.cpp

```diff
@@ -553,10 +553,13 @@ bool PluginCall(uint8_t Function, struct EventStruct *event, String& str)
         const int freemem_begin = ESP.getFreeHeap();
         #endif
 
-        PluginCallForTask(taskIndex, Function, &TempEvent, str, event);
+        bool retval = PluginCallForTask(taskIndex, Function, &TempEvent, str, event);
 
-        #ifndef BUILD_NO_DEBUG
         if (Function == PLUGIN_INIT) {
+          if (!retval) {
+            Settings.TaskDeviceEnabled[taskIndex] = false; // Initialization failed: Disable plugin!
```

Comment:
Have all plugins been checked to return `success = true;`  during `PLUGIN_INIT` ?

Reply:
Yes, all validated, except any open PRs, other then my own.

Reply:
I just validated the open PRs that introduce new plugins, and all return their state as intended from `PLUGIN_INIT`. 👍